pkgs/by-name/vi/vimb-unwrapped/package.nix

```diff
@@ -3,7 +3,6 @@
   stdenv,
   fetchFromGitHub,
   pkg-config,
-  libsoup_2_4,
```

Comment:
This is still an [explicit dependency](https://github.com/fanglingsu/vimb/blob/eade0be96e38674244866a25fcf90c7a08c48fe4/src/main.c#L26) so we generally prefer to include it explicitly, even though it is unlikely webkitgtk would stop propagating it.

Reply:
wait what? I am somewhat surprised this package ever worked then.... Fixing.

Reply:
libsoup is propagated by webkit so from the point of vimb, `libsoup_3` was present.